llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -795,6 +795,53 @@ struct AllocMatchInfo {
   bool Matched = false;
 };
 
+DenseMap<uint64_t, SmallVector<CallEdgeTy, 0>>
+memprof::extractCallsFromIR(Module &M) {
+  DenseMap<uint64_t, SmallVector<CallEdgeTy, 0>> Calls;
+
+  auto GetOffset = [](const DILocation *DIL) {
+    return (DIL->getLine() - DIL->getScope()->getSubprogram()->getLine()) &
```

Comment:
Should we assert that getLine() (for the call and the function start) are non-zero?

Reply:
For now, let's keep this identical to another instance of `GetOffset` in `MemProfiler.cpp`.